### PR TITLE
Update callback plugin

### PR DESCRIPTION
**Pull request by mgirgisf**
Update custom_logger to generate summary file called "test_run_result.out" with the number of failures and success and then generate a detailed report for each host in separate file called "{host}_results.log"

### CODE REVIEW

File: callback_plugins/custom_logger.py [hunk 1]
```diff
@@ -1,72 +1,69 @@
...
-    def log(self, host, category):
-         # only interested in the test tasks, not setup or debug
-         if self.current_task.startswith("RHELOSP"):
-
-             if category == "failed":
-                self.failed +=1
-             if category == "passed":
-                 self.passed +=1
-
-             test_result_message = self.MSG_FORMAT.format(task=self.current_task, status=category)
```
Comment (elfiesmelfie) [inline callback_plugins/custom_logger.py:56]: There's a requirement on the format of the results.out file to remain consistent.
The format needs to be "{task_name}={passed|failed}"

Comment (elfiesmelfie) [reply callback_plugins/custom_logger.py:56]: The format should be {testID}={passed|failed}

Existing users expect the test_run_result.out file to contain results in this format.

Comment (mgirgisf) [reply callback_plugins/custom_logger.py:56]: I kept the file "test_run_result.out" using the same old format , I just updated the functions to be more organised and we will have another file called "{host}_results.log" that will have more details so we can use it in the zuul job.

**Review by lnatapov — APPROVED** [review]

**Comment by mgirgisf** [comment]
**Depends-On:** #90

**Comment by elfiesmelfie** [comment]
> **Depends-On:** #90

The Depends-On needs to be in the PR description -- I've updated it.
I don't know whether the GHA tests will run or not, since it's Zuul that acts on Depends-On.

**Review by elfiesmelfie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**